make/tests

```diff
@@ -208,3 +208,7 @@ endif
 # Include the test dependencies
 ##
 include make/test-math-dependencies
+
+.PHONY: test 
```

Comment:
I think you missed the point.

Remove the `test` target completely. Not just the line with `.PHONY`